packages/esm-form-entry-app/src/app/form-schema/form-schema.service.ts

```diff
@@ -80,6 +133,72 @@ export class FormSchemaService {
     return formSchema;
   }
 
+  private getFormTranslationsByUuidFromServer(formUuid: string) {
+    const formTranslations: ReplaySubject<any> = new ReplaySubject(1);
+    this.fetchFormTranslationUsingFormMetadata(formUuid).subscribe(
+      (translations: any) => {
```

Comment:
Type here should **not** be `any`.

Reply:
Explicit types help us catch programming errors, especially when other people need to use your code. Using `any` where it's not called for (i.e., the object can literally be anything) makes maintenance harder and pretty well guarantees that we will add bugs. If you don't know the type, set it to `unknown`, but in this case, we seem to know the type...